src/components/Dialogs/Dialogs.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+import style from './Dialogs.module.css'
+import { NavLink } from 'react-router-dom';
```

Comment:
This import is not grouped properly, move it to the correct place in this file.

Reply:
/fxd